lib/errors.js

```diff
@@ -174,7 +174,7 @@ module.exports = function(config) {
 `,
 	migrateConfig: () => `error 12:
 Please migrate your configuration:
-https://github.com/SAP/karma-ui5/blob/master/docs/migrate-v1.md`,
```

Comment:
We should maybe keep a `master` branch with the relevant docs files and link to main from there. Otherwise the links will stop working.

Reply:
when using the branch renaming feature, the link will be forwarded to main. So, there should be no need to keep a master branch

Reply:
Ah cool, I forgot about that 👍🏻